src/test/java/unstable/ContextMenuUnstableTests.java

```diff
@@ -0,0 +1,97 @@
+
+    private static final String PANEL_IDENTIFIER = "#dummy/dummy_col0";
+
+    @Before
+    public void setup() {
+        Platform.runLater(stage::show);
+        Platform.runLater(stage::requestFocus);
+
+        FilterTextField filterTextField = find(PANEL_IDENTIFIER + "_filterTextField");
+        filterTextField.setText("");
+        Platform.runLater(filterTextField::requestFocus);
+
+        click(PANEL_IDENTIFIER + "_filterTextField");
+        push(KeyCode.ENTER);
+    }
+
+    /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_twelveIssuesInListView_issuesCorrectlyMarkedReadUnread()
+            throws NoSuchFieldException, IllegalAccessException {
+
+        ListPanel issuePanel = find(PANEL_IDENTIFIER);
+        Field listViewField = ListPanel.class.getDeclaredField("listView");
+        listViewField.setAccessible(true);
+        IssueListView listViewValue = (IssueListView) listViewField.get(issuePanel);
+
+        // scrolls to the end of the panel
+        listViewValue.scrollAndShow(12);
+
+
+        //checking for issue #7 and below twice to make sure issues are marked correctly for read/unread cases
+        clickAndMarkIssuesBelow(issuePanel, 7, true);
+        verifyReadStatusOfIssuesBelow(issuePanel, 7, true);
+        clickAndMarkIssuesBelow(issuePanel, 7, true);
+        verifyReadStatusOfIssuesBelow(issuePanel, 7, true);
+        clickAndMarkIssuesBelow(issuePanel, 7, false);
+        verifyReadStatusOfIssuesBelow(issuePanel, 7, false);
+        clickAndMarkIssuesBelow(issuePanel, 7, false);
+        verifyReadStatusOfIssuesBelow(issuePanel, 7, false);
+
+        //checking for the last issue to ensure correct marking of issues on/below as read/unread when no issues below
+        clickAndMarkIssuesBelow(issuePanel, 1, true);
+        verifyReadStatusOfIssuesBelow(issuePanel, 1, true);
+        clickAndMarkIssuesBelow(issuePanel, 1, false);
+        verifyReadStatusOfIssuesBelow(issuePanel, 1, false);
+    }
+
+    /**
+     * Marks issues in the panel from Issue index in the UI right to the end of the list as read/unread
+     * @param isMarkAsRead If true, marks the selected issue and below as read
+     *                     If false, marks the selected issue and below as unread
+     * @param index The issue number in the panel
+     */
+    private void clickAndMarkIssuesBelow(ListPanel issuePanel, int index, boolean isMarkAsRead){
+        click("#" + issuePanel.getId() + "_" + index);
+        rightClick("#" + issuePanel.getId() + "_" + index);
+        ContextMenu contextMenu = issuePanel.getContextMenu();
+        for (MenuItem menuItem : contextMenu.getItems()){
+            awaitCondition(menuItem::isVisible);
+        }
+        if (isMarkAsRead){
+            click("Mark all below as read");
+        } else {
+            click("Mark all below as unread");
+        }
```

Comment:
1. Can use the Ternary operator?
2. Use constants from the relevant class instead of using magic strings (need to make them public first)?